packages/user-interface/src/pages/task/task-page.tsx

```diff
@@ -100,63 +86,55 @@ const TaskPage = () => {
     }
   };
 
-  const onFormSubmit = (formioSubmission: any) => {
+  const onFormSubmit = async (formioSubmission: any) => {
     if (formioSubmission?.state === 'submitted') {
-      completeTask(formioSubmission.data);
+      await submitTask({
+        variables: {
+          id: `${taskId}`,
+          submission: formioSubmission.data,
+        },
+      });
     }
   };
 
-  const redrawForm = (form: any) => {
-    form.triggerRedraw();
-  };
+  if (loading) {
+    return null;
+  }
 
-  const removeLocalStorage = () => {
-    localStorage.removeItem(formId);
-  };
+  if (!formDefinitionId && !formDefinitionUrl) {
+    return <Alert variant="error" title={intl.formatMessage({id: 'task.fetchError'})} text="" />;
+  }
 
-  const getSubmittedMessage = () => (
-    <React.Fragment>
-      <h2 className="utrecht-heading-2 utrecht-heading-2--distanced pb-1">Taak is afgerond</h2>
-      <Link
-        onClick={removeLocalStorage}
-        className="btn btn-primary"
-        role="button"
-        rel="noopener noreferrer"
-        to="/taken"
-      >
-        Klik hier om terug te gaan naar je openstaande taken
-      </Link>
-    </React.Fragment>
-  );
+  if (submitted) {
+    return (
+      <Alert
+        variant="success"
+        title={intl.formatMessage({id: 'task.completeTitle'})}
+        text={intl.formatMessage({id: 'task.completeDescription'})}
+      />
+    );
+  }
 
   return (
     <React.Fragment>
       <Helmet>
+        {/* https://docs-portal.valtimo.nl/form.io-related/missing-form.io-icons */}
         <link
           rel="stylesheet"
           href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
```

Comment:
@Borai mogen we runtime in een applicatie achter digid inlog gebruik maken van dit CDN ivm security?

Reply:
@petervanmanenritense Ik had gehoopt dat jij/jullie hier een antwoord op zouden hebben. Er stonden nog meer import script. Die heb ik kunnen vervangen voor npm packages. Echter in jullie documentatie (zie comment regel 121) staat ook dat dit script nodig is. 

Zie ook commit: [https://github.com/nl-portal/nl-portal-frontend-libraries/commit/5f308af833fd4482bf9af6ec59551a5e9e28ec8b#diff-5118aba05550e5[…]92f09a305950d330d3cb](https://github.com/nl-portal/nl-portal-frontend-libraries/commit/5f308af833fd4482bf9af6ec59551a5e9e28ec8b#diff-5118aba05550e55d83d3569807c81e24f26e5d835b2c92f09a305950d330d3cb)

Heb dit nog even dubbel gecheckt bij Klaas maar die kon mij ook niet helemaal vertellen of die font awesome lib echt nodig was. Dus op basis van jullie docs heb ik hem voor nu laten staan.

Reply:
@Borai terechte opmerking maar dit zijn mijns inziens wel 2 verschillende vraagstukken.
1. Hebben we dit echt nodig
2. Als we het nodig hebben mogen we het dan op een cdn van deze club gebruiken
Het antwoord op vraag 1 zouden we eenvoudig moeten kunnen testen
Volgens mij is het antwoord op vraag 2 nee; dus dan zouden we de lib in onze repo kunnen opnemen?

Reply:
@petervanmanenritense True, maar in die zin wel afhankelijk. Zou mooi zijn als we font awesome niet nodig zouden hebben. Heb jij een form definition waarbij er gebruik word gemaakt van icons?

Reply:
Ik stel voor dat we dit probleem voor nu even negeren, en op een later moment oppakken

Reply:
@petervanmanenritense Heb het toch nog kunnen testen met een voorbeeld form. En font awesome v4 npm package lost het probleem op.